doc/api/crypto.md

```diff
@@ -2371,6 +2378,7 @@ changes:
      required only if the `format` is `'der'` and ignored if it is `'pem'`.
   * `passphrase`: {string | Buffer} The passphrase to use for decryption.
   * `encoding`: {string} The string encoding to use when `key` is a string.
+  * `asymmetricKeyType` {string} The requested asymmetric key type.
```

Comment:
It would be good to document the expected values here (or link to where they are documented)